airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -181,13 +181,15 @@ def __normalize_identifier_name(
             return f"'{result}'"
         return result
 
-    def apply_quote(self, input: str) -> str:
+    def apply_quote(self, input: str, literal=True) -> str:
+        if literal:
```

Comment:
not sure I understand this change - previously all of our quotes were using `quote` or `adapter.quote`; why do we need to switch them to just wrapping in single-quote marks?

Reply:
yes I agree not very obvious

before my change this function generated only such output:

```
adapter.quote('table_name')
```

as you know on  jinja2 "level" it means: pass String literal `'table_name'` to jinaj2 function `adapter.quote`

but I have a new requirement I need to get:

```
adapter.quote(this.schema)
```

on jinja2 level it means pass jinja2 **variable** to jinja2 function.

Reply:
ah that makes sense (also I misread the code, thought it was doing a `return f"'{input}'"`). So basically this is behaving exactly the same for `literal=False`.